docs/web_apps.rst

```diff
@@ -106,7 +106,7 @@ Not all features have all of these pieces:
 Example: Registration from Case List
 ====================================
 
-As an example, consider `registration from the case list <https://confluence.dimagi.com/display/commcarepublic/Minimize+Duplicates+Method+1%3A+Registration+From+the+Case+List>`_:
+As an example, consider `registration from the case list <https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143951603/Case+List+and+Case+Detail+Configuration>`_:
```

Comment:
More specific: https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143951603/Case+List+and+Case+Detail+Configuration#Minimize-Duplicates%3A-Registration-From-the-Case-List